docs/usage/nfd-master.md

```diff
@@ -15,7 +15,6 @@ extended resources based on requests from nfd-workers and 3rd party extensions.
 
 ## NodeFeature controller
 
-**EXPERIMENTAL**
```

Comment:
Please revisit his whole section (or the whole file) with thought and drop mentions about enabling the API with `-enable-nodefeature-api`. You could note in the end that the API can be disabled with `-enable-nodefeature-api=false` in which case NFD will fallback to the old gRPC API.

Also in the "NodeFeatureRule controller" section below there is text that talks about gRPC being the default so that needs to be changed, too